### PR TITLE
Adds 4chanterelle and 8chanterelle

### PR DESCRIPTION
**Pull request by Iamgoofball**
the contents of the mushrooms accurately reflect the contents of their actual counterparts

### CODE REVIEW

**Comment by TZK13** [comment]
Cute, :+1:

**Comment by paprka** [comment]
i laughed harder than i should have

**Comment by AlexCrimson** [comment]
DAE hate 4chan? lol

**Comment by Miauw62** [comment]
ayy lmao

**Comment by babinsquared** [comment]
:thumbsup: for giving players a way to create a traitor-only poison. Venom is one of the more interesting poisons available.

File: code/modules/hydroponics/grown.dm [hunk 1]
```diff
@@ -963,6 +963,29 @@ obj/item/weapon/reagent_containers/food/snacks/grown/shell/eggy/add_juice()
...
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/fourchanterelle/add_juice()
+	..()
+	reagents.add_reagent("toxin", 1 + round((potency / 25), 1))
+	bitesize = 1 + round(reagents.total_volume / 2, 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/eightchanterelle
+	seed = /obj/item/seeds/eightchantermycelium
+	name = "8chanterelle cluster"
+	desc = "<I>8Cantharellus Viltoxarius</I>: These purple vile toxic little shrooms sure look like they'll rend your brain apart!"
```
Comment (stapler2025) [inline code/modules/hydroponics/grown.dm]: 8Can

Comment (phil235) [inline code/modules/hydroponics/grown.dm]: this isn't a typo, normal chanterelle is "Cantharellus" in its desc.

Comment (TZK13) [inline code/modules/hydroponics/grown.dm]: It does sort of ruin the pun though.

**Comment by Razharas** [comment]
We need a reference police here to evaluate this pr

**Comment by paprka** [comment]
:police_car: WEE WOO WEE WOO :police_car: 

This one isn't really that bad but I'd rather not have 8chan since we're named after a 4chan board and don't want people to get butthurt about 8chan and stuff, it's needless drama even for a simple reference. 4chanterelle can stay but if you really want to keep 8chan I'd rather they both not enter the game imo.

**Comment by Iamgoofball** [comment]
if people start getting in drama arguments about ethics in spaceman journalism over a joke plant name they need to step back and look at what they are doing with their life

**Comment by Kromgar** [comment]
But Paprka its about ethics in coding

**Comment by paprka** [comment]
ethics in shitposting

**Comment by dannno** [comment]
yeah if people get fucked up over a plant joke fuck 'em

File: code/modules/hydroponics/grown.dm [hunk 2]
```diff
@@ -963,6 +963,29 @@ obj/item/weapon/reagent_containers/food/snacks/grown/shell/eggy/add_juice()
...
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/fourchanterelle/add_juice()
+	..()
+	reagents.add_reagent("toxin", 1 + round((potency / 25), 1))
+	bitesize = 1 + round(reagents.total_volume / 2, 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/eightchanterelle
+	seed = /obj/item/seeds/eightchantermycelium
+	name = "8chanterelle cluster"
+	desc = "<I>8Chantharellus Viltoxarius</I>: These purple vile toxic little shrooms sure look like they'll rend your brain apart!"
```
Comment (Tenebrosity) [inline code/modules/hydroponics/grown.dm]: Pls make the c in chan lowercase

File: code/modules/hydroponics/grown.dm [hunk 3]
```diff
@@ -963,6 +963,29 @@ obj/item/weapon/reagent_containers/food/snacks/grown/shell/eggy/add_juice()
 	reagents.add_reagent("nutriment", 1 + round((potency / 25), 1))
 	bitesize = 1 + round(reagents.total_volume / 2, 1)
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/fourchanterelle
+	seed = /obj/item/seeds/fourchantermycelium
+	name = "4chanterelle cluster"
+	desc = "<I>4Chantharellus Toxarius</I>: These green toxic little shrooms sure look vile!"
```
Comment (Tenebrosity) [inline code/modules/hydroponics/grown.dm]: Pls make the c in chan lowercase

File: code/modules/hydroponics/seeds.dm [hunk 4]
```diff
@@ -546,6 +546,42 @@
...
 	growthstages = 3
 	plant_type = 2
+	mutatelist = list(/obj/item/seeds/fourchantermycelium)
+
+/obj/item/seeds/fourchantermycelium
+	name = "pack of 4chanterelle mycelium"
+	desc = "This mycelium grows into 4chanterelle mushrooms."
+	icon_state = "mycelium-4chanter"
+	species = "4chanter"
+	plantname = "4Chanterelle Mushrooms"
```
Comment (Tenebrosity) [inline code/modules/hydroponics/seeds.dm]: Pls

File: code/modules/hydroponics/seeds.dm [hunk 5]
```diff
@@ -546,6 +546,42 @@
...
+	growthstages = 3
+	plant_type = 2
+	mutatelist = list(/obj/item/seeds/eightchantermycelium)
+
+/obj/item/seeds/eightchantermycelium
+	name = "pack of 8chanterelle mycelium"
+	desc = "This mycelium grows into 8chanterelle mushrooms."
+	icon_state = "mycelium-8chanter"
+	species = "8chanter"
+	plantname = "8Chanterelle Mushrooms"
```
Comment (Tenebrosity) [inline code/modules/hydroponics/seeds.dm]: P

**Comment by Stickymayhem** [comment]
Could this not be slightly subtler? Like quadchanterelle and octchaneterelle?

Descriptions are fine but almost any other name would be better. A good sprite would be best.

**Comment by Dorsisdwarf** [comment]
The joke is that when you say it it sounds like 4chan. 

It would have to be "fourchanterelle" to still be funny. (It isn't funny.)

**Comment by Cheridan** [comment]
le gimmicks
also merge conflicts anyway

**Pull request closed without merging**